buildSrc/src/integTest/groovy/org/opensearch/gradle/fixtures/DistributionDownloadFixture.groovy

```diff
@@ -65,7 +65,7 @@ class DistributionDownloadFixture {
     private static String urlPath(String version, OpenSearchDistribution.Platform platform) {
         String fileType = ((platform == OpenSearchDistribution.Platform.LINUX ||
                 platform == OpenSearchDistribution.Platform.DARWIN)) ? "tar.gz" : "zip"
-        "/downloads/opensearch/opensearch-${version}-${platform}-x86_64.$fileType"
+        "/releases/core/opensearch/${version}/opensearch-${version}-${platform}-x86_64.$fileType"
```

Comment:
@saratvemulapalli this will only handle the opensearch distribution, how about the ES distribution?